.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "greengrass_parsec_workshop/parsec-client-java"]
```

Comment:
Why do we need this submodule? It's not used anywhere
`aws-greengrass-parsec-provider` submodule already includes `parsec-client-java` as a submodule where it's used for build